src/main/java/com/ge/predix/uaa/token/lib/DefaultZoneConfiguration.java

```diff
@@ -25,17 +25,16 @@
 import org.springframework.util.AntPathMatcher;
 
 public class DefaultZoneConfiguration implements InitializingBean {
-    private String trustedIssuerId;
     private List<String> allowedUriPatterns;
     private List<String> trustedIssuerIds = new ArrayList<>();
 
+    // This field is for backward compatibility.
     public String getTrustedIssuerId() {
-        return this.trustedIssuerId;
+        return this.trustedIssuerIds.isEmpty() ? null : this.trustedIssuerIds.get(0);
```

Comment:
Add tests for setting trusted issuer(s) with null and empty list and getting trusted issuer(s) when not configured.

Reply:
thanks Irina, i added the tests. to be fair those should've been there to begin with :)